src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -1173,7 +1174,7 @@ namespace QgsWms
     {
       bool validLayer = false;
       bool queryableLayer = true;
-      for ( QgsMapLayer *layer : qgis::as_const( layers ) )
+      for ( QgsMapLayer *layer : layers )
```

Comment:
Was that changed on purpose?
https://github.com/qgis/QGIS/pull/31157/files

Reply:
nope, I didn't change it at all, because I did this work locally before your commit https://github.com/qgis/QGIS/commit/6fd35f6f1769534f3c5b70a4ee7ebb46eb6477c8, I should have rebased.